sorc/ncep_post.fd/BNDLYR.f

```diff
@@ -286,6 +289,17 @@ SUBROUTINE BNDLYR(PBND,TBND,QBND,RHBND,UBND,VBND,       &
               END IF 
               WBND(I,J,LBND)     = WBND(I,J,LBND)*RPSUM
               QCNVBND(I,J,LBND)  = QCNVBND(I,J,LBND)*RPSUM
```

Comment:
@JesseMeng-NOAA From line 279 to 291, variable T,Q,UH and VH are not used, why adding check in line 277 and 288?

Reply:
This block performs averaging boundary layer variables TBND, QBND, UBND, VBND, etc, those were computed from T, Q, UH, VH in the blocks above. Here we do a validation of valid grid points before transform the variables to model gridtypes.

Reply:
Thanks for the explanation.